bigbluebutton-html5/tests/webdriverio/INSTALL.md

```diff
@@ -0,0 +1,95 @@
+# Acceptance Testing in HTML Client. Getting Started
+
+The test suite for HTML5 client is currently under active development. The following instructions will help you install all the necessary tools and libraries to run the exiting specs and start writing your own tests.
+
+## Run Selenium Server
+
+Assuming that you have the BigBlueButton repository in `/home/firstuser/dev`, navigate to the directory containing acceptance tests:
+```sh
+$ cd /home/firstuser/dev/bigbluebutton/bigbluebutton-html5/tests/webdriverio
+```
+
+Create `tools` folder to store various third-party binaries:
+```sh
+$ mkdir tools
```

Comment:
Could you add an empty file there, so that you could commit this folder and avoid this step?
As far as I know a common practice is to add an empty file called `.gitkeep`.